tests/classic/sdp_c/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
this test should go under tests/bluetooth....

Reply:
Yes. But there is an issue that the path will exceed the maximum full path of CMAKE.
There is a warning,
```
CMake Warning in C:/lylezhu/zephyrproject/zephyr/boards/nxp/mimxrt1170_evk/CMakeLists.txt:
  The object file directory

    C:/lylezhu/zephyrproject/zephyr/twister-out/mimxrt1170_evk@B_mimxrt1176_cm7/tests/bluetooth/classic/sdp_c/sdp.c/zephyr/boards/nxp/mimxrt1170_evk/CMakeFiles/boards__nxp__mimxrt1170_evk.dir/./

  has 190 characters.  The maximum full path to an object file is 250
  characters (see CMAKE_OBJECT_PATH_MAX).  Object file

    b088242880d0c7ed6f5663b9dea6f973/evkbmimxrt1170_flexspi_nor_config.c.obj

  cannot be safely placed under this directory.  The build may not work
  correctly.
```
And the building issue is,
```
FAILED: zephyr/boards/nxp/mimxrt1170_evk/CMakeFiles/boards__nxp__mimxrt1170_evk.dir/b088242880d0c7ed6f5663b9dea6f973/evkbmimxrt1170_flexspi_nor_config.c.obj 

C:\lylezhu\zephyr-sdk-0.16.4\arm-zephyr-eabi\bin\arm-zephyr-eabi-gcc.exe -DBOARD_FLASH_SIZE=CONFIG_FLASH_SIZE*1024 -DCACHE_MODE_WRITE_THROUGH -DCPU_MIMXRT1176DVMAA_cm7 -DKERNEL -DK_HEAP_MEM_POOL_SIZE=0 -DPICOLIBC_LONG_LONG_PRINTF_SCANF -DTC_RUNID=419819eda10292bd79bde6adccba3cf8 -DXIP_BOOT_HEADER_ENABLE=1 -DXIP_BOOT_HEADER_XMCD_ENABLE=1 -DXIP_EXTERNAL_FLASH -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR__=1 -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/boards/evkbmimxrt1170/xip -IC:/lylezhu/zephyrproject/zephyr/twister-out/mimxrt1170_evk@B_mimxrt1176_cm7/tests/bluetooth/classic/sdp_c/sdp.c/zephyr/include/generated/zephyr -IC:/lylezhu/zephyrproject/zephyr/include -IC:/lylezhu/zephyrproject/zephyr/twister-out/mimxrt1170_evk@B_mimxrt1176_cm7/tests/bluetooth/classic/sdp_c/sdp.c/zephyr/include/generated -IC:/lylezhu/zephyrproject/zephyr/soc/nxp/imxrt -IC:/lylezhu/zephyrproject/zephyr/lib/posix/options/getopt -IC:/lylezhu/zephyrproject/zephyr/soc/nxp/imxrt/imxrt11xx/. -IC:/lylezhu/zephyrproject/zephyr/soc/nxp/imxrt/. -IC:/lylezhu/zephyrproject/zephyr/soc/nxp/imxrt/imxrt11xx -IC:/lylezhu/zephyrproject/zephyr/subsys/testsuite/include -IC:/lylezhu/zephyrproject/zephyr/subsys/testsuite/coverage -IC:/lylezhu/zephyrproject/zephyr/subsys/testsuite/ztest/include -IC:/lylezhu/zephyrproject/zephyr/subsys/bluetooth -IC:/lylezhu/zephyrproject/zephyr/subsys/shell/modules/kernel_service/thread/.. -IC:/lylezhu/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -IC:/lylezhu/zephyrproject/zephyr/modules/cmsis/. -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/MIMXRT1176 -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/MIMXRT1176/drivers -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/MIMXRT1176/xip -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/common -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/caam -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/lpuart -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/igpio -IC:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/cache/armv7-m7 -IC:/lylezhu/zephyrproject/zephyr/modules/hal_nxp/. -isystem C:/lylezhu/zephyrproject/zephyr/lib/libc/common/include -Wshadow -fno-strict-aliasing -Werror -Os -imacros C:/lylezhu/zephyrproject/zephyr/twister-out/mimxrt1170_evk@B_mimxrt1176_cm7/tests/bluetooth/classic/sdp_c/sdp.c/zephyr/include/generated/zephyr/autoconf.h -fno-printf-return-value -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m7 -mthumb -mabi=aapcs -mfp16-format=ieee -mtp=soft --sysroot=C:/lylezhu/zephyr-sdk-0.16.4/arm-zephyr-eabi/arm-zephyr-eabi -imacros C:/lylezhu/zephyrproject/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wdouble-promotion -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -fno-asynchronous-unwind-tables -ftls-model=local-exec -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=C:/lylezhu/zephyrproject/zephyr/tests/bluetooth/classic/sdp_c=CMAKE_SOURCE_DIR -fmacro-prefix-map=C:/lylezhu/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=C:/lylezhu/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -specs=picolibc.specs -D_POSIX_THREADS -std=c99 -MD -MT zephyr/boards/nxp/mimxrt1170_evk/CMakeFiles/boards__nxp__mimxrt1170_evk.dir/b088242880d0c7ed6f5663b9dea6f973/evkbmimxrt1170_flexspi_nor_config.c.obj -MF zephyr\boards\nxp\mimxrt1170_evk\CMakeFiles\boards__nxp__mimxrt1170_evk.dir\b088242880d0c7ed6f5663b9dea6f973\evkbmimxrt1170_flexspi_nor_config.c.obj.d -o zephyr/boards/nxp/mimxrt1170_evk/CMakeFiles/boards__nxp__mimxrt1170_evk.dir/b088242880d0c7ed6f5663b9dea6f973/evkbmimxrt1170_flexspi_nor_config.c.obj -c C:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/boards/evkbmimxrt1170/xip/evkbmimxrt1170_flexspi_nor_config.c

C:/lylezhu/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/boards/evkbmimxrt1170/xip/evkbmimxrt1170_flexspi_nor_config.c:89:1: fatal error: opening dependency file zephyr\boards\nxp\mimxrt1170_evk\CMakeFiles\boards__nxp__mimxrt1170_evk.dir\b088242880d0c7ed6f5663b9dea6f973\evkbmimxrt1170_flexspi_nor_config.c.obj.d: No such file or directory

   89 | };

      | ^

compilation terminated.
```

If I move the folder from `tests/bluetooth/classic/sdp_c` to `tests/classic/sdp_c`, there is not such issue. Or , for path `tests/bluetooth/classic/sdp_c`, the option `-O sdp_c` for `west twister` command line can be used to shorten the output path of twister. I tried to add `extra_configs: - -O sdp_c` or `extra_args: - -O sdp_c` to `classic\sdp_c\testcase.yaml`. But It cannot work.

So I move the folder from `tests/bluetooth/classic/sdp_c` to `tests/classic/sdp_c`.

Is there any way can be used to fix the issue?